### PR TITLE
Edit saving data section in ug

### PR DESCRIPTION
**Pull request by elaineshijie**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1266,15 +1266,18 @@ testers are expected to do more *exploratory* testing.
 ### Saving data
 
 1. Dealing with missing/corrupted data files
-    1. If a missing file is detected, the system will automatically create a .json file.
-   2.  If a corrupted file is detected (i.e. missing any key properties), the system will delete the corrupted file, and replace it with a new one.
+   1. If a missing file is detected, the system will automatically create a .json file.
+   2. If a corrupted file is detected (i.e. missing any key properties), the system will delete the corrupted file, and replace it with a new one.
```
Comment (HollaG) [inline docs/DeveloperGuide.md]: Can just write a new blank file

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -1266,15 +1266,18 @@ testers are expected to do more *exploratory* testing.
 ### Saving data
 
 1. Dealing with missing/corrupted data files
-    1. If a missing file is detected, the system will automatically create a .json file.
-   2.  If a corrupted file is detected (i.e. missing any key properties), the system will delete the corrupted file, and replace it with a new one.
+   1. If a missing file is detected, the system will automatically create a .json file.
```
Comment (HollaG) [inline docs/DeveloperGuide.md]: and fill it with sample data

Comment (elaineshijie) [reply docs/DeveloperGuide.md]: changed in the recent commit, thank you

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (elaineshijie) [reply docs/DeveloperGuide.md]: changed in the recent commit, thank you

**Review by pzl111 — APPROVED** [review]
LGTM

**Pull request merged**